### PR TITLE
Wordy approach

### PR DESCRIPTION
**Pull request by bobahop**
Approach for Wordy. I either don't like or don't grok other approaches.

### CODE REVIEW

File: exercises/practice/wordy/.approaches/dunder-getattribute/content.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+def answer(question):
+    question = question.removeprefix("What is").removesuffix("?").strip()
+    if not question: raise ValueError("syntax error")
+    if question.isdigit(): return int(question)
+
+    foundOp = False
+    for name, op in OPS.items():
+        if name in question:
+            question = question.replace(name, op)
+            foundOp = True
```
Comment (BethanyG) [inline exercises/practice/wordy/.approaches/dunder-getattribute/content.md:21]:
```suggestion
            found_op = True
```

File: exercises/practice/wordy/.approaches/dunder-getattribute/content.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    if not foundOp: raise ValueError("unknown operation")
```
Comment (BethanyG) [inline exercises/practice/wordy/.approaches/dunder-getattribute/content.md:22]:
```suggestion
    if not found_op: raise ValueError("unknown operation")
```

File: exercises/practice/wordy/.approaches/dunder-getattribute/content.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    "divided by": "__truediv__"
+}
+
+
+def answer(question):
+    question = question.removeprefix("What is").removesuffix("?").strip()
+    if not question: raise ValueError("syntax error")
+    if question.isdigit(): return int(question)
+
+    foundOp = False
```
Comment (BethanyG) [inline exercises/practice/wordy/.approaches/dunder-getattribute/content.md:17]:
```suggestion
    found_op = False
```

File: exercises/practice/wordy/.approaches/introduction.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    "divided by": "__truediv__"
+}
+
+
+def answer(question):
+    question = question.removeprefix("What is").removesuffix("?").strip()
+    if not question: raise ValueError("syntax error")
+    if question.isdigit(): return int(question)
+
+    foundOp = False
```
Comment (BethanyG) [inline exercises/practice/wordy/.approaches/introduction.md:34]:
```suggestion
    found_op = False
```

File: exercises/practice/wordy/.approaches/introduction.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    for name, op in OPS.items():
+        if name in question:
+            question = question.replace(name, op)
+            foundOp = True
```
Comment (BethanyG) [inline exercises/practice/wordy/.approaches/introduction.md:38]:
```suggestion
            found_op = True
```

File: exercises/practice/wordy/.approaches/introduction.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    if not foundOp: raise ValueError("unknown operation")
```
Comment (BethanyG) [inline exercises/practice/wordy/.approaches/introduction.md:39]:
```suggestion
    if not found_op: raise ValueError("unknown operation")
```

**Review by BethanyG — APPROVED** [review]
Fixing stray `foundOp`.  Converted to `found_op`.

Otherwise, LGTM.

**Comment by bobahop** [comment]
Thanks for catching the naming!

**Pull request merged**